openjdk11-sbt/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM adoptopenjdk/openjdk11:jdk-11.0.7_10
+
+LABEL version="jdk-11.0.7_sbt-1.3.10"
+LABEL maintainer="tsuru@chatwork.com"
+
+SHELL ["/bin/bash", "-c"]
+
+ENV PATH=$PATH:/root/.sdkman/candidates/sbt/current/bin:/opt/java/openjdk/bin
+
+RUN apt update && \
+  apt-get install unzip && \
+  apt-get install zip && \
+  curl -s https://get.sdkman.io | bash && \
+  source /root/.sdkman/bin/sdkman-init.sh && \
+  sdk install sbt 1.3.10 && \
```

Comment:
@imahiro-t 
Do you need to install scala?
Installing scala and sbt on the image will make the build and test faster.

Reply:
@cw-ozaki 
Since the version of Scala depends on the project, we will not install it here and absorb it with sbt.